client/src/pages/Scheduler.js

```diff
@@ -64,95 +67,100 @@ const Scheduler = () => {
         duration: event.duration,
         link: event.link,
       });
+
       setAppointmentDetails({ ...appointmentDetails, eventId: event._id });
+      setEventActive(event.active);
```

Comment:
Could you use the eventDetails state and add an active key to it?

Reply:
Yes, that's valid. But I think for simplicity I will keep it as is unless there is a performance issue.